lib/Corres_Method.thy

```diff
@@ -180,7 +180,7 @@ lemmas corres_split_liftE_bindE [corres_splits] =
    that weak guard, the rule is safe. This enables us to solve trivial cases without adding
    unsafe rules to the [corres] set. *)
 lemmas [corres_term] =
-  corres_return_eq_same corres_gets_trivial select_corres_eq
+  corres_return_eq_same_unit corres_return_eq_same corres_gets_trivial select_corres_eq
```

Comment:
Thanks for adding that (and moving the `corres_machine_op_Id` lemmas). That should make `corres` nicer to use in a whole bunch of situations.